source/common/router/router.cc

```diff
@@ -720,7 +720,10 @@ void Filter::handleNon5xxResponseHeaders(const Http::HeaderMap& headers,
   }
 }
 
-void Filter::onUpstream100ContinueHeaders(Http::HeaderMapPtr&& headers) {
+void Filter::onUpstream100ContinueHeaders(Http::HeaderMapPtr&& headers,
+                                          UpstreamRequest& upstream_request) {
+  const uint64_t response_code = Http::Utility::getResponseStatus(*headers);
```

Comment:
Do we need `response_code`, or can we assume it to be `100` at this point?

Reply:
I was trying to future-proof for 1xx but it looks like we're pretty strict elsewhere that it's exactly 100 so I'll go with that :-)

Reply:
Thanks for the clarification. :)